src/core/binding.c

```diff
@@ -1432,12 +1432,8 @@ QuicBindingDeliverDatagrams(
         // Only Initial (version specific) packets are processed from here on.
         //
         switch (Packet->Invariant->LONG_HDR.Version) {
-        case QUIC_VERSION_DRAFT_27:
```

Comment:
http didn't depend on any of these versions, right? We won't have to make changes in there by removing these?

Reply:
Correct. They have no QUIC version specific code.